pa

```diff
@@ -99,7 +99,8 @@ pw_list() {
 }
 
 git_add_and_commit() {
-    if $git_enabled; then git add "$1" && git commit -qm "$2"; fi
+    git add "$1" || die "failed to git add $1"
```

Comment:
style nits:

- "failed to" -> "Couldn't"
- `die` should be on its own line
- there should be an empty line in-between add and commit

Reply:
nits addressed in 861d1c9